packages/core/test/builtin.spec.js

```diff
@@ -79,8 +79,8 @@ test('builtin - paths soft-bindings preserve "this" but allow override', async (
 
     defineTwo: () => {
       const { Buffer } = require('buffer')
-      const thisChecker = require('thisChecker')
-      const { SomeClass } = require('someClass')
+      const thisChecker = require('abc')
+      const { SomeClass } = require('xyz')
```

Comment:
if we use consistent names, we can configure `dependency-cruiser` to automatically ignore them, which is what I'm doing here.